contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -218,7 +218,8 @@ def testConvertMesh( self ) :
 		cubeMesh = cube.readObject( 0.0 )
 
 		self.assertTrue( isinstance( cubeMesh, IECoreScene.MeshPrimitive ) )
-		self.assertEqual( set( cubeMesh.keys() ), { "P", "uv", "Cs" } )
+		self.assertEqual( set( cubeMesh.keys() ), { "P", "uv" } )
+		self.assertTrue( "render:displayColor" in cube.attributeNames() )
```

Comment:
Hmm, I'd failed to anticipate this change in behaviour. I wonder if we should rename on load so that we end up with an attribute called `render:Cs` in Cortex, and do the reverse conversion on write? I'll try to find out if anyone is relying on the current behaviour...

Reply:
It seems that another difference is that we're loading as `Color3fVectorData` rather than `Color3fData`.

Reply:
Yeah, I think it'd be pretty poor to lose viewport colouring for USD assets as part of this update. We weren't accounting for a `Cs` attribute in IECoreGL, but I've opened a PR to add it : https://github.com/ImageEngine/cortex/pull/1208. I've gone for just supporting `Cs` is that PR (rather than `displayColor` etc), as I want this second round of USD attribute support to be as non-disruptive as possible in Gaffer.

Reply:
Although Cinesite have indicated they're not concerned with back-compatibility for `Cs` at this point, so loading a `render:displayColor` attribute could be OK, as long as we support it in GL. I'm in two minds. In terms of what's easiest to implement, it's probably `displayColor` in GL...any opinions?